gymnasium/vector/async_vector_env.py

```diff
@@ -250,7 +250,7 @@ def np_random(self) -> tuple[np.random.Generator, ...]:
     def reset(
         self,
         *,
-        seed: int | list[int] | None = None,
+        seed: int | list[int | None] | None = None,
```

Comment:
This could also be typed as `int | list[int] | list[None]| None` if we want to soft-disallow specifying seeds for a subset of envs.

Reply:
No I think that's fine